cheatsheets_draft/Secrets_Management_CheatSheet.md

```diff
@@ -401,33 +401,107 @@ EaaS is a model in which users subscribe to a cloud-based encryption service wit
 - Key handling and cryptographic implementations are taken care of by Encryption Service, not by developers
 - The provider could add more services to interact with the sensitive data
 
-## 9 Workflow in Case of Compromise
-
-(by @thatsjet)
-
-### 9.1 Process
-
-## 11 Secret detection
-
-(by @thatsjet)
-
-- Many native integrations possible (Cloud platforms, CI/CD tooling, application libraries, container orchestrators)
-- Secret lifecycle (rotation, deletion, lifespan)
-- Key material management (keys to the kingdom)
-- Open source? (Depending on security posture)
-- Encryption (at rest, in transit)
-- Access control (fine-grained)
-- Performance
-- Audit logs
-- Scalable (enterprise)
-- Manageable operations (upgrading, recovery)
-- Agnostic
-- Support for many secrets backends: database, certificates, ssh keys, cloud providers, key/value, etc
-- Dynamic secrets
-- Encryption as a service
-- Fine-grained policies (MFA requirements)
-- Extensibility
-- Documentation
+## 8 Detection
```

Comment:
Could you update the index up top to match this title?

Reply:
Will do in a next iteration :)